apps/cowswap-frontend/src/common/updaters/FinalizeTxUpdater.tsx

```diff
@@ -175,11 +176,17 @@ function finalizeOnChainCancellation(
   orderId: string,
   sellTokenSymbol: string
 ) {
-  const { chainId, isSafeWallet, dispatch, addPopup, cancelOrdersBatch } = params
+  const { chainId, isSafeWallet, dispatch, addPopup, cancelOrdersBatch, account } = params
 
   if (receipt.status === 1) {
     // If cancellation succeeded, mark order as cancelled
     cancelOrdersBatch({ chainId, ids: [orderId], isSafeWallet })
+
+    emitCancelledOrderEvent({
```

Comment:
I think in other PRs you killed redux? (maybe I'm making things up). 

Anyways, its was not required in the PR to kill redux, but just I saw we emit the event, and also we trigger the redux action. Disregard this comment, as its just an observation and I prefer to not overcomplicate the addition of events right now

Reply:
I've killed only a part of redux, more specifically `popupMiddleware`.
We still emit redux actions here to change orders state.
I would love to get rid of it as well, but it will take awhile :)

Reply:
Yes, I understand 👍